packages/site/pages/components/icon.js

```diff
@@ -78,7 +78,11 @@ export default _ => (
       <Code language="bash">
         npm install @pluralsight/ps-design-system-icon
       </Code>
-
+      <P>
+        In your package webpack config in order to enable treeshaking add the
+        following config option:
+      </P>
+      <Code>{`optimization: { usedExports: true }`}</Code>
```

Comment:
Won't teams usually be using `mode: "production"` instead, and that would enable the tree shaking optimization?

see: https://webpack.js.org/guides/tree-shaking/#conclusion

Reply:
True I should reference if they want tree shaking in dev non-prod mode. I make that change.